ui/apps/ui/src/app/collections/repositories/types.ts

```diff
@@ -98,6 +98,7 @@ export interface IFilterConfig {
   type: 'multiselect' | 'select' | 'date' | 'tag' | 'range';
 
   onFacetsFetch?: (bucketValues: IFacetBucket[]) => IFilterNode[]; // !!! only for multiselect !!!
+  defaultCollapsed?: boolean;
```

Comment:
I think it introduces issues later to allow booleans to have 3 values (`true`, `false` and `undefined`).
It would be better to restructure code to accept only boolean.

But this is a minor. It can stay "as is"

Reply:
done